rpms/awips2.edex/Installer.edex/programs/updateNDM.pl

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/perl
+
+#This script gets installed with EDEX and a cronjob is setup to run this script daily. This script initially sets up a local git repo and is configured to just checkout the National Dataset files from Unidata's public git repo and then copies the files to /awips2/edex/data/ndm to keep the databases updated with new metar/station information.
+
+$ndmGitPath = "rpms/awips2.edex/Installer.edex/ndm/";
+
+#get the current branch/version of awips2
+$version = `grep AWIPSII_VERSION /awips2/repo/awips2/build/buildEnvironment.sh`;
+chomp $version;
+@split = split(/"/,$version);
+
+$branch = "unidata_$split[1]";
+
+#Check if ndm location has been created
+$ndmDir="/awips2/dev/ndm";
+if(!-d $ndmDir)
+{
+  `mkdir $ndmDir`;
+}
+
```

Comment:
Since both the if and after the if, cd to the $ndmDir, I think you should jsut do that here, around line 20, and then remove it at line 24 and 36

Reply:
Unfortunately, perl is not like bash, where you can do it once. Every time a new set of commands are run (in perl this is by using the back ticks `) it's like it would open a new terminal in the home directory.

Reply:
Oooh. Okay I see the surrounding \` \` now.